rotkehlchen/chain/ethereum/manager.py

```diff
@@ -306,26 +273,29 @@ def default_call_order(self, skip_etherscan: bool = False) -> List[NodeName]:
         """
         result = []
         if NodeName.OWN in self.web3_mapping:
-            result.append(NodeName.OWN)
+            # Assigning one is just a default since we always use it
+            # The weight is only important for the other nodes since they
+            # are selected suing this paramenter
```

Comment:
![200](https://user-images.githubusercontent.com/1658405/173350046-6cf73d6a-b41e-4730-8ebe-b1616d3b867d.gif)


```suggestion
            # are selected using this parameter
```